parent/pom.xml

```
@@ -29,17 +29,17 @@
         <java.minimum.version>1.8</java.minimum.version>
 
         <!-- core -->
-        <commons-lang3.version>3.4</commons-lang3.version>
+        <commons-lang3.version>3.9</commons-lang3.version>
         <javax-inject.version>1</javax-inject.version>
 
         <!-- module dependencies -->
-        <jackson.version>2.9.4</jackson.version>
-        <guice.version>4.1.0</guice.version>
-        <argparse4j.version>0.4.4</argparse4j.version>
+        <jackson.version>2.10.2</jackson.version>
+        <guice.version>4.2.2</guice.version>
+        <argparse4j.version>0.8.1</argparse4j.version>
 
         <!-- TEST -->
-        <guava.version>19.0</guava.version>
-        <testng.version>6.8.21</testng.version>
+        <guava.version>28.2-jre</guava.version>
```

Guava 28.2 is also the first version released in 2020. I suggest using 28.0. 
```suggestion
        <guava.version>28.0-jre</guava.version>
```